ci/incremental/setup_conda_environment.cmd

```diff
@@ -11,11 +11,10 @@ call deactivate
 @rem Display root environment (for debugging)
 conda list
 @rem Clean up any left-over from a previous build
-conda remove --all -q -y -n %CONDA_ENV%
+conda remove --all -q -y -n pandas-dev
 @rem Scipy, CFFI, jinja2 and IPython are optional dependencies, but exercised in the test suite
-conda env create -n %CONDA_ENV% --file=ci\deps\azure-windows-%CONDA_PY%.yaml
+conda env create -n pandas-dev --file=ci\deps\azure-windows-%CONDA_PY%.yaml
```

Comment:
you don't need the `-n pandas-dev`, the name of the environment is already defined in the yaml file